SesameIT/jizo/ingest/parser.yml

```diff
@@ -0,0 +1,61 @@
+name: jizo
+pipeline:
+  - name: json_log
+    description: ""
+    filter: ""
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+
+  - name: parsed_date
+    description: ""
+    filter: ""
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_log.message.timestamp}}"
+        output_field: datetime
+        format: null
+        timezone: UTC
+
+  - name: set_base_fields
+
+stages:
+  set_base_fields:
+    actions:
+      - set:
+          source.ip: "{{json_log.message.src_ip}}"
+          "@timestamp": "{{parsed_date.datetime}}"
+          destination.ip: "{{json_log.message.dest_ip}}"
+          destination.port: "{{json_log.message.dest_port}}"
+          source.port: "{{json_log.message.src_port}}"
+          network.protocol: "{{json_log.message.app_proto}}"
+          network.transport: "{{json_log.message.proto}}"
+          action.name: "{{json_log.message.alert.action}}"
+          action.type: "{{json_log.message.event_type}}"
```

Comment:
If the `event_type` is `alert`, we suggest you to set `event.kind: "alert"`

Reply:
done

Reply:
All forwarded events will have `event_type = "alert"` or the event_type matches the [expected values](https://www.elastic.co/guide/en/ecs/8.11/ecs-allowed-values-event-kind.html) from the ECS specification?

Otherwise, We suggest you to add a new `set` action, that define the `event.kind` field, with a filter about the value of `event_type`.

```yaml
      - set:
          event.kind: "alert"
        filter: "{{json_message.message.event_type == 'alert'}}"
```

Reply:
Yes, all forwarded events will have event_type = "alert". Our product jizo always sends logs with event_type as alert. Thus, we can keep it like this, without a filter.